set.mm

```diff
@@ -464135,6 +464135,17 @@ Class abstractions (a.k.a. class builders)
       FUMQABCTUAUBUCUDUE $.
   $}
 
+  ${
+    $d A x y z $.  $d ph z $.
+    opabssi.1 $e |- ( ph -> <. x , y >. e. A ) $.
+    $( Sufficient hypothesis for a collection of ordered pairs to be a subclass
+       of a relation.  (Contributed by Peter Mazsa, 21-Oct-2019.) $)
+    opabssi $p |- { <. x , y >. | ph } C_ A $=
```

Comment:
1. I see, great:) 2. No way. I insist on this addition at least : (Revised by Thierry Arnoux, 18-Feb-2022.)

Reply:
I think it's ok as is, but the credit could as well go to @tirix, that's not very important in this case.  The general view is that "ideas are cheap, implementation is hard", so credits should generally go to the person who does the formalization over the one who has the idea of doing it.

Reply:
@benjub I completely agree. @tirix please correct this.

Reply:
@tirix I mean please correct this in a way that you Thierry is the author.

Reply:
Ok, since it's my choice I kept the version with contributed/revised. 😁